src/phpDocumentor/ConfigurationFactory.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+namespace phpDocumentor;
+
+final class ConfigurationFactory
+{
+    /**
+     * @var Uri
+     */
+    private $uri;
+
+    /**
+     * @var string
+     */
+    private $schemaPath;
+
+    /**
+     * @param Uri    $uri
+     * @param string $schemaPath
+     */
+    public function __construct(Uri $uri, $schemaPath = '../../data/xsd/phpdoc.xsd')
```

Comment:
down below i use ```$this->schemaPath``` as follows: ```$dom->schemaValidate(__DIR__ . DIRECTORY_SEPARATOR . $this->schemaPath);```
If that is not correct, I can change it in the constructor